acquire/collector.py

```diff
@@ -331,13 +350,40 @@ def collect_glob(self, pattern: str, module_name: Optional[str] = None) -> None:
             else:
                 self.report.add_glob_collected(module_name, pattern)
 
+    def collect_symlink(
+        self,
+        path: Path,
+        module_name: Optional[str] = None,
+        follow: bool = True,
+        volatile: bool = False,
+    ) -> None:
+        try:
+            outpath = self.create_outpath(str(path))
+            entry = path.get()
```

Comment:
Now this code path is the only one actually passing in a `FilesystemEntry`, all other code paths (including the memory dump one) are passing in a `Path`-like object (either a real `Path` or a `TargetPath`).